NonEmergencyAmbulanceTransportation/R4/resources/Questionnaire-R4-NonEmergencyAmbulanceTransportationProgressNote.json

```diff
@@ -0,0 +1,497 @@
+            {
+              "type": "boolean",
+              "code": [],
+              "required": false,
+              "linkId": "8.7.1",
+              "text": "Danger to self or others"
+            },
+            {
+              "type": "boolean",
+              "code": [],
+              "required": false,
+              "linkId": "8.7.2",
+              "text": "Confused, combative, lethargic, comatose"
+            }
+          ]
+        },
+        {
+          "type": "boolean",
+          "code": [],
+          "required": false,
+          "linkId": "8.8",
+          "text": "Other"
+        },
+        {
+          "type": "text",
+          "code": [],
+          "required": false,
+          "linkId": "8.9",
+          "text": "Describe",
+          "enableWhen": [
+            {
+              "answerBoolean": true,
+              "question": "8.8",
+              "operator": "="
+            }
+          ]
+        },
+        {
+          "type": "group",
+          "linkId": "8.10",
+          "code": [],
+          "required": false,
+          "text": "Required services that necessitate abulance transport",
+          "item": [
+            {
+              "type": "boolean",
+              "code": [],
+              "required": true,
+              "linkId": "8.10.1",
+              "text": "Are there services that cannot be provided in the patient’s current setting?"
+            },
+            {
+              "type": "open-choice",
+              "required": false,
+              "repeats": true,
+              "linkId": "8.10.2",
+              "text": "If Yes, specify by selecting one or more of the following:",
+              "enableWhen": [
+                {
+                  "answerBoolean": true,
+                  "question": "8.10.1",
+                  "operator": "="
+                }
+              ],
+              "answerOption": [
+                {
+                  "valueCoding": {
+                    "code": "Dialysis"
+                  }
+                },
+                {
+                  "valueCoding": {
+                    "code": "Wound care"
+                  }
+                },
+                {
+                  "valueCoding": {
+                    "code": "Radiation therapy"
+                  }
+                },
+                {
+                  "valueCoding": {
+                    "code": "Chemotherapy"
+                  }
+                },
+                {
+                  "valueCoding": {
+                    "code": "O&P services"
+                  }
+                },
+                {
+                  "valueCoding": {
+                    "code": "Imaging"
+                  }
+                },
+                {
+                  "valueCoding": {
+                    "code": "Outpatient therpay"
+                  }
+                }
+              ]
+            },
+            {
+              "type": "text",
+              "code": [],
```

Comment:
you don't need 8.10.3 since open-choice accepts free text input

Reply:
Removed it and pushed the change.